mt940/models.py

```diff
@@ -37,6 +37,38 @@ def __new__(cls, *args, **kwargs):
             # No need to do anything special there :)
             return datetime.date.__new__(cls, *args, **kwargs)
 
+class DateTime(datetime.datetime, Model):
```

Comment:
We could but I would vote against it since it could break existing implementations. We could make them use the same base class however to reuse the code :)